ulauncher/modes/extensions/extension_remote.py

```diff
@@ -173,55 +147,79 @@ def download(self, commit_hash: str | None = None, warn_if_overwrite: bool = Fal
                         rmtree(self._dir)
                     move(tmp_dir, self._dir)
             commit_timestamp = getmtime(self._dir)
+            return commit_hash, commit_timestamp
```

Comment:
Changed to early returns instead of big if/elif statements, because the latter can get confusing and/or break more easily.